first pass at converting FOSDEM 2018 pentabarf XML

garrett opened the pull request:
This has updated LDAP info as well, for RH name matching.

(Source code changes have been pushed in the OSAS GitLab instance.)

dneary commented on 2018/FOSDEM.yml:
```diff
@@ -11,3 +7,1137 @@ description: |
+    favorite programming language as the first language to learn
+    programming.
+
+- title: 'Deploying OpenStack in the opnv community: cross community ci'
```
This should be OPNFV (upper case) - I corrected the typo in the talk title in Penta, so this may update automatically. Also, CI should be caps.

garrett replied:
This data comes directly from FOSDEM… it should be fixed there.

That said, I can add it to the rewrite list and send a new PR with our local rewrite fix.

dneary replied:
Thanks - I fixed it on the FOSDEM site (well, the typo - the capitalization was right there already, but is not reroduced here for some reason). I was not sure whether changes on fosdem.org would be reflected automatically here.

garrett replied:
Well, I'll pull down the XML again at some point and re-run the script and send a new PR — and the updates should show up. The changes are broken into 3 regular expressions that uniquely fix the errors, so it would just not apply the fixes if they're not needed. It'll work out. :wink:

FOSDEM typically seems to update the XML export a few times a week... and it doesn't seem to immediately update.

garrett commented:
I've adjusted the script and updated the PR. It's ready for squashing and merging, unless any other issues are found.

(Naturally, it could be squash/merged and a follow-up PR can be sent. Just let me know via an issue on GitHub mentioning my name if that's the case.)

garrett commented:
BTW: Here are the results of the change: https://github.com/OSAS/rh-events/pull/346/commits/edb7badd02d5a7e03aa73532f9f0c5a933bf4e76

dneary commented:
Who pulls these merge requests typically?

rbowen commented:
@bproffitt I believe.

nixpanic commented:
[Gluster-4.0 and GD2](https://fosdem.org/2018/schedule/event/gluster4_and_gd2/) is presented by Red Hatter @kshlm. Could you add that talk as well?

garrett commented:
@nixpanic: I've updated the info and added @kshlm.

Time's running out to merge things in; FOSDEM is 3 & 4 of Feb (this weekend).
